README.md

```diff
@@ -139,21 +141,33 @@ Read the [**contribution guide**](.github/CONTRIBUTING.md) for more detailed inf
 </details>
 
 <details>
-  <summary>I tried to login via Access Token & OTP but it does not work?</summary>
+  <summary>I tried to login via Access Token & OTP but why isn't it working?</summary>
   <p>You can't login via Access Token & OTP all together due to the lifetime of the OTP code, you'll be required to login in every few seconds.</p>
 </details>
 
 <details>
-  <summary>Why my Private Repo Wiki does not show up?</summary>
+  <summary>Why are my Private Repo and Enterprise Wiki not showing up?</summary>
   <p>It's due to FastHub scraping GitHub Wiki page & Private Repos require session token that FastHub doesn't have.</p>
 </details>
 
 <details>
-  <summary>I login with Enterprise account but can't interact with anything other than my Enterprise GitHub</summary>
-  <p>Well, logically, you can't access anything else other than your Enterprise, but FastHub made that possible but can't do much about it,
-  in most cases since your login credential doesn't exists in GitHub server. But in <b>few</b>
-  cases your GitHub account Oauth token will do the trick.</p>
+  <summary>I login with Enterprise account but can't interact with anything other than my Enterprise GitHub.</summary>
+  <p>Well, logically, you can't access anything else other than your Enterprise, but FastHub made that possible but can't do much about it, in most cases since your login credential doesn't exists in GitHub server. But in <b>few</b> cases your GitHub account Oauth token will do the trick.</p>
+</details>
+
+<details>
+  <summary>Why am I having problems editing Issues/PRs?</summary>
+  <p>If you are editing a public Org repo, then please contact your Org to grant access to FastHub or use Access Token to login!.</p>
```

Comment:
Over here. Change the text to "If you are unable to edit an **issue** in a public...."

Reply:
Ok!